examples/nat_scapy_tests.py

```diff
@@ -144,9 +144,11 @@ def run_server2(serverport=12022, natport=35002):
     ackn += 1
     print "Replying with Ack"
     send(ip/TCP(sport=serverport,dport=natport,flags="A", seq=seqn,ack=ackn))
+    time.sleep(1)
 
     ## Should now be unable to use the connection
     # Send something
+    time.sleep(1)
     seqn += 1
     ackn += 1
```

Comment:
seqn and ackn are supposed to be the sequence number and the ack number for TCP. The information is not checked, so at the moment it's pretty pointless.